cirq/optimizers/eject_z.py

```diff
@@ -17,6 +17,7 @@
 from typing import Optional, cast, TYPE_CHECKING, Iterable
 from collections import defaultdict
 import sympy
+import numpy as np
```

Comment:
import numpy as np should be before import sympy

Reply:
did this change, but my local pylint keeps giving me a `RecursionError` so I'm pushing and crossing my fingers that that was the only problem